### DISCUSSION TITLE
Flex Conversational Insights - Frame AI

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#3877] [Office hours] [answered]
On behalf of Carol @cdpestitschek

### Background

Allowing supervisors to see the breakdown of information, helps them to understand and investigate the reasoning behind the main KPI and scoring that is currently selected. The breakdown of information supports the underlying data which provides value in what they are seeing.

Each KPI has their respective charts and data, it means, when selected it changes all the charts and data below.

**Use case**
As a Supervisor, I want to be able to break down the data I'm visualizing, so that I can understand the reasoning behind the KPI, as well as why and where I should act.

**Default status**
By default one of the KPIs will be presented as ‘selected’, offering value (the data) without initial effort, as well as, working to reduce the learning curve. Showing that there are 2 different statuses, or rather, that you can expect to be able to click on the others as well. 

Example of Widget on hover - Feature findability

<img width="1640" alt="Screenshot 2024-04-26 at 13 28 02" src="https://github.com/twilio-labs/paste/assets/49287701/f5eb3de3-1805-4bdd-9bc3-531c954dac98">

Example of Widget when selected

<img width="1898" alt="Screenshot 2024-04-26 at 13 28 58" src="https://github.com/twilio-labs/paste/assets/49287701/198cda82-30f0-46f5-b2ae-0d8fe5ed7bbf">

**Use case**
As a Supervisor, I want to have clarity about which KPI is currently selected so that I can ensure I am visualizing and analyzing the expected data.

### Feedback

**I would like feedback and guidance on:** 
– Is this the best way to promote this drilldown experience?
– Is there another component rather than “Cards”, that should be considered for this purpose?
– In future, considering the “on hover status”, could tooltips be of a concern within the widget?
– The findability. Is it clear that Supervisors will be able to drilldown once they click on a widget card?
– Is this navigation accessible?

### Stage of design

High-fidelity designs. Engineering are in build.

### Research
[Supervisor Frame AI research](https://app.heymarvin.com/insight/3e3f11c7-d2e9-4ed2-83de-193956623c4f/?u=1705334567473&u=1714136156311)

_No response_

### Relevant links

[Figma designs](https://www.figma.com/file/9kev68PgqCOkjFQxE6K0ec/Flex---Supervisor-Conversational-Insights?type=design&node-id=10451%3A144477&mode=design&t=78QsIJN85KozlR3u-1)

[UX Review Presentation](https://docs.google.com/presentation/d/1_R4ehCqhN7xpzXo36bMdJ_JcvatU1Q57b-KJ7jD_MXI/edit#slide=id.g2a6ab572694_0_0)
Slide 19, 20 and 21 

### Are you looking for visual design support?

- [X] Yes

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours @cdpestitschek! [Here's the recording.](https://twilio.zoom.us/rec/share/-FIRYDYhweUS_XPFKaPWw0_am-k_9JjzH5S3zlmAswiPGiVTZVhVQ-sXoK-XDjRd.8A2EHe0rqWDaU5eG) You can find the password in slack.
